archunit/src/test/java/com/tngtech/archunit/lang/syntax/elements/GivenClassesThatTest.java

```diff
@@ -469,6 +471,8 @@ public void areNotMetaAnnotatedWith_predicate() {
 
     @Test
     public void implement_type() {
+        ArchConfiguration.get().setProperty(VERIFY_INPUT_NOT_EMPTY_PROPERTY_NAME, Boolean.FALSE.toString());
```

Comment:
What about just setting this to `false` for the whole test class? Optimally just by using `ArchConfigurationRule` so the configuration is automatically reset after each test. I think for these tests it's okay to switch the default behavior for the whole class, that reduces clutter.

Reply:
Yes, for this class I agree that it is a good solution.

However, I'm not sure If I understood you correctly regarding the `ArchConfigurationRule`. The rule we use to clean up the config after each test, right? But to set our property to `false` we need to use a `@Before` or can this be done also by using the rule?

With `@Before` (and `@After` inside the rule) we are changing the configuration on each test. Would it in this case not be better to use a `@BeforeClass`/`@AfterClass` so that it is only changed once for the test?